### PR TITLE
Remove Yuni Strings in solver/ [ANT-1198]

### PR DESCRIPTION
**Pull request by payetvin**

### CODE REVIEW

File: src/solver/constraints-builder/cbuilder.cpp [hunk 1]
```diff
@@ -392,11 +392,6 @@ bool CBuilder::completeCBuilderFromFile(const String& filename)
                     key = p->key;
                     key.toLower();
 
-                    if (key == "study")
-                    {
-                        pStudyFolder = p->value;
-                        continue;
-                    }
```
Comment (flomnes) [inline src/solver/constraints-builder/cbuilder.cpp:399]: Why remove this ?

File: src/ui/simulator/application/main/logs.cpp [hunk 2]
```diff
@@ -206,9 +206,9 @@ void ApplWnd::connectLogCallback()
     logs.callback.connect(this, &ApplWnd::onLogMessageDeferred);
 }
 
-void ApplWnd::onLogMessageDeferred(int level, const String& message)
+void ApplWnd::onLogMessageDeferred(int level, const std::string& message)
 {
-    if (not message.empty() and message.first() != '[')
+    if (not message.empty())
```
Comment (flomnes) [inline src/ui/simulator/application/main/logs.cpp]: Behavior changes

File: src/solver/constraints-builder/cbuilder.cpp (hunk 1 above)
Comment (payetvin) [reply src/solver/constraints-builder/cbuilder.cpp:399]: variable `pStudyFolder` is unused
on this note we should decide what to do with the constraint builder as it's not used by antares web right now

File: src/ui/simulator/application/main/logs.cpp (hunk 2 above)
Comment (payetvin) [reply src/ui/simulator/application/main/logs.cpp]: didn't bother bc it's logs in the GUI

done

**Review by flomnes — APPROVED** [review]

**Pull request merged**